### ISSUE TITLE
Webpage code injection

### ISSUE THREAD

**Issue by NickClark** [rocksdanister/lively#1284]
**Is your feature request related to a problem? Please describe.**
I was looking to use Lively as a way to display a third party map on my desktop. Unfortunately the default zoom and position isn't right by default and the site doesn't provide any query strings that will work. I found that I could easily inject some code to do what I wanted.

**Describe the solution you'd like**
It would be nice if there was some way to do some code injection on a site `onLoad`.

**Describe alternatives you've considered**
I tried making my own html file that had an iframe to house the site, but it has iframe headers set to prevent this

**Comment by ecr-oha-lime** [comment]
这个想法听起来不错

**Comment by ecr-oha-lime** [comment]
@rocksdanister

**Comment by rocksdanister** [comment]
Yes its a good idea.. can also be used for hiding scrollbar in certain pages.

**Comment by ecr-oha-lime** [comment]
我之前是通过保存网页的方式来改变要添加到库的网页的。
==========translate===========
I used to change the web pages to add to the library by saving the web pages.
